ufersa/plataforma-sabia
# Add service default thumbnails based on type

## Summary

Resolves #882 

## QA Steps

1. Access home page and search page
2. Assert that services has correct thumbnails based on type

(check [here](https://www.figma.com/file/EN9rw1NbspVx3IRgT3SiJL/App-Plataforma-Sabi%C3%A1?node-id=695%3A442))

## Checklist
